hs_core/authentication.py

```diff
@@ -5,9 +5,9 @@
 from django.urls import reverse, resolve
 from django.conf import settings
 from django.utils.http import urlencode
-from django.contrib.auth.models import User
 from rest_framework.authentication import BaseAuthentication
 from keycloak.keycloak_openid import KeycloakOpenID
+from theme.utils import get_user_from_username_or_email
```

Comment:
I'm having second thoughts about the utils idea. We currently don't have any imports to theme in hs_core (and we also don't have any imports of hs_core into theme).  I think I would rather have theme importing hs_core than the other way around. 

I'm also looking at the rest_framework import and realize that wasn't great of me to put this rest_framework authentication class in hs_core. I don't want this patch to be held up by refactoring and we need to get this patch out soon so.

I'll write a tech debt ticket to capture these concerns and to help inform a later refactor and approve this ticket.

Reply:
good call